overlord/devicestate/devicestate_test.go

```diff
@@ -357,25 +359,16 @@ func (s *deviceMgrSuite) TestFullDeviceRegistrationHappyWithProxy(c *C) {
 		Model: "pc",
 	})
 
-	// avoid full seeding
-	s.seeding()
-
-	// not started without gadget
-	s.state.Unlock()
-	s.se.Ensure()
-	s.state.Lock()
-
-	becomeOperational := s.findBecomeOperationalChange()
```

Comment:
Do we lose testing here? Or was/is this unnecessary?

Reply:
this is the changes in 5430055  , the distinction is not necessary anymore because we always wait for seeded if there's  gadget (which is always the case on Core )